guibot/calibrator.py

```diff
@@ -56,14 +58,14 @@ class Calibrator(object):
     multiple random starts from a uniform or normal probability distribution.
     """
 
-    def __init__(self, needle=None, haystack=None, config=None):
+    def __init__(self, needle: Target = None, haystack: Image = None,
+                 config: bool = None) -> None:
         """
         Build a calibrator object for a given match case.
 
-        :param haystack: image to look in
-        :type haystack: :py:class:`target.Image` or None
         :param needle: target to look for
-        :type needle: :py:class:`target.Target` or None
+        :param haystack: image to look in
+        :param config:
```

Comment:
This has empty description, let's use "config file for calibration"

Reply:
Done